data_steward/cdr_cleaner/cleaning_rules/valid_death_dates.py

```diff
@@ -35,9 +35,24 @@
 # or after the current date.
 SANDBOX_INVALID_DEATH_DATE_ROWS = common.JINJA_ENV.from_string("""
 CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{sandbox_table}}` AS (
-SELECT *
-FROM `{{project_id}}.{{dataset_id}}.{{table}}`
-WHERE {{table}}_date < '{{program_start_date}}' OR {{table}}_date > {{current_date}})
+SELECT d.*
+FROM `{{project_id}}.{{dataset_id}}.{{table}}` d
+LEFT JOIN (
+        SELECT
+            person_id, MIN(o.observation_date) first_ppi_date
```

Comment:
This looks like it will only find the earliest date.  If I interpret 1643 correctly, we want to sandbox any death records that exist before any PPI records.  Added the following test code to check the current implementation when the death date falls between PPI records.  It fails with the existing implementation.
[1643_patch.txt](https://github.com/all-of-us/curation/files/6580336/1643_patch.txt)

Reply:
Sounds good, will make that change.

Reply:
Made the change. I think just flipping the MIN to a MAX did the trick.